Cargo.toml

```diff
@@ -1,32 +1,7 @@
-[package]
-name = "centrifuge-chain"
-version = "0.10.35"
-authors = ["Centrifuge <admin@centrifuge.io>"]
-description = "Centrifuge chain implementation in Rust."
-build = "build.rs"
-edition = "2021"
-license = "LGPL-3.0"
-default-run = "centrifuge-chain"
-homepage = "https://centrifuge.io/"
-repository = "https://github.com/centrifuge/centrifuge-chain"
-
-[package.metadata.wasm-pack.profile.release]
-# `wasm-opt` has some problems on linux, see
-# https://github.com/rustwasm/wasm-pack/issues/781 etc.
-wasm-opt = false
-
-[badges]
-travis-ci = { repository = "centrifuge/centrifuge-chain", branch = "master" }
-maintenance = { status = "actively-developed" }
-is-it-maintained-issue-resolution = { repository = "centrifuge/centrifuge-chain" }
-is-it-maintained-open-issues = { repository = "centrifuge/centrifuge-chain" }
```

Comment:
I think this should stay in the root Cargo.toml as it is not related to the client but this repository.

Reply:
Sadly, you can not place `[badges]` in `Cargo.toml` that only contains workspace related things. They seem to be related to a specific crate

Reply:
I see, then this can be marked as resolved.